tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Copy LDAP-Client config
+  copy:
+    src: ldap.conf
+    dest: /etc/ldap/ldap.conf
+    owner: root
+    group: root
+    mode: "u=rw,g=r,o=r"
```

Comment:
Why not the decimal notiation?

Reply:
You're right, this stems from a wierd template.